crates/cli/src/lib.rs

```diff
@@ -593,7 +613,7 @@ pub fn build(
 
                     // No need to waste time freeing this memory,
                     // since the process is about to exit anyway.
-                    std::mem::forget(arena);
+                    // std::mem::forget(arena);
```

Comment:
Is this intentional/necessary? If so, we should probably delete the whole comment.